tests/unit/Hooks/ImageHooksTest.php

```diff
@@ -112,7 +117,7 @@ public function testPostCreate() {
 			->method('insert')
 			->with($this->imageDimension);
 
-		$this->imagehooks->post_create($jpgFile);
+		$this->mockImageHooks->postCreate($jpgFile);
```

Comment:
No, actually that was intentional, as I had mentioned that when I am using imagehooks it is giving a segfault but it doesn't give a segfault when I used the mock version.